src/include/distributed/multi_logical_replication.h

```diff
@@ -26,8 +26,40 @@ extern void LogicallyReplicateShards(List *shardList, char *sourceNodeName,
 									 int sourceNodePort, char *targetNodeName,
 									 int targetNodePort);
 
+extern XLogRecPtr GetRemoteLogPosition(MultiConnection *connection);
+extern List * GetQueryResultStringList(MultiConnection *connection, char *query);
+
+extern void DropShardSubscription(MultiConnection *connection,
+								  char *subscriptionName);
+extern void DropShardPublication(MultiConnection *connection, char *publicationName);
+
+extern void DropShardUser(MultiConnection *connection, char *username);
+extern void DropShardMoveReplicationSlot(MultiConnection *connection,
+										 char *publicationName);
+
+
+extern char * ShardSubscriptionRole(Oid ownerId, char *operationPrefix);
```

Comment:
nitpick: Reverse parameter order as that is the order they are used for
formatting.

Reply:
Same for ShardSubscriptionName

Reply:
The order is fine as at couple of places in the existing code its used in the same way. Kept consistent with.
extern void DropShardUser(MultiConnection *connection, char *username);